README.md

```diff
@@ -2,24 +2,250 @@
+
+Discussion of the PR will continue until all dissenting opinions and questions have been addressed, upon which the PR may be put forward for Provisional acceptance. Note that addressing dissenting opinions does not necessarily mean achieving complete resolution on them. A dissent is not considered to have "veto" power over a proposal, but must be addressed to a degree that convinces the majority of ASC members.
+
+#### Provisional acceptance
+The process for acceptance to Provisional status begins when the author(s) assign an "**RFC**" label to the GitHub PR, add an empty "**Straw Poll**" comment to it, and send an announcement of the PR to the PMIx community mailing list requesting consideration of the PR for Provisional status. The announcement shall include a link to the PR, a brief description of the proposed change, and indicate any desired timeline for approval. It is especially important that proposals tied to contractual milestones be so designated to ensure that members are alerted to potential time pressures.
+
+The "Straw Poll" comment is used by the ASC as a mechanism for obtaining a sense of how many people have viewed the PR and are following the discussion. This serves no official purpose other than to help define participation in the consensus building process and record the following tally:
+
+* Those participants that approve the proposal must indicate that by placing a :+1: emoji on the comment.
+
+* Those participants that have specific concerns regarding the proposal must indicate that by placing a :-1: emoji on the comment.
+
+Anyone indicating concern must provide a comment on the ticket following the "**Straw Poll**" comment describing their concern. Sufficient detail should be provided to allow the author(s) to respond to the specific concerns. The author(s) can then work to address the concern, modifying the PR as required - modification should be accompanied by an appropriate comment citing the original participant who raised the concern to ensure they are notified of the change. Participants shall indicate their acceptance or rejection of the modification by so marking the change comment and adjusting their original "vote" on the straw poll if appropriate.
+
+The ASC Co-Chairs shall monitor the status of discussion on the PR. Once discussion appears to have consolidated (either measured by participation in the straw poll, rate of comments, or a combination of both), the Co-Chairs will schedule the proposal for a reading at the next quarterly meeting of the ASC. The proposal shall be included in the announced agenda for the meeting along with a link to the PR and the results of the straw poll.
```

Comment:
I'd also advocate for the proposal authors to have an active role here. If the control to bring forward a proposal is only under the discretion of the chairs (and the chairs say don't pay attention or whatever), progress could be stalled. Can authors request to schedule a reading? Is there a reason why the chairs would deny the request? (assuming the rest of the procedures have been followed)

Reply:
> Once discussion appears to have consolidated (either measured by participation in the straw poll, rate of comments, or a combination of both), or at the request of the proposal's author(s), the Co-Chairs will schedule the proposal for a reading at the next quarterly meeting of the ASC.